inc/functions/litespeed.php

```diff
@@ -0,0 +1,42 @@
+<?php
+use WP_Rocket\Logger\Logger;
+
+defined( 'ABSPATH' ) || die( 'Cheatin&#8217; uh?' );
+
+/**
+ * Purge Litespeed URL
+ *
+ * @since 3.4.1
+ *
+ * @param  string $url The URL to purge.
+ * @return void
+ *
+ * @author Soponar Cristina
+ */
+function rocket_litespeed_header_purge_url( $url ) {
+    $parse_url      = get_rocket_parse_url( $url );
+	$path           = rtrim($parse_url['path'], '/');
+    $private_prefix = 'X-LiteSpeed-Purge: ' . $path ;
+
+    Logger::debug( 'X-LiteSpeed', [
```

Comment:
```suggestion
	if ( headers_sent() ) {
		Logger::debug( 'X-LiteSpeed: headers already sent', [
			'rocket_litespeed_header_purge_url',
			'path' => $private_prefix,
		] );
		return;
	}

	Logger::debug( 'X-LiteSpeed', [
```

Reply:
If you don't use `$private_prefix` in the `if` condition, then it should be moved at the beginning of the function: there is no need to parse + rtrim before it then.